### ISSUE TITLE
Issue Menu Management After update to 5.4.0.6

### ISSUE THREAD

**Issue by mahfoudfx** [crocodic-studio/crudbooster#759]
After the update to 5.4.0.6, there is clearly issue when access Menu Management
 (6/6) ErrorException
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'cms_privileges_id' in 'field list' (SQL: select `cms_privileges_id` from `cms_menus_privileges` where `cms_menus_id` is null) (View: C:\laragon\www\rnd\vendor\crocodicstudio\crudbooster\src\views\default\type_components\select2\component.blade.php) (View: C:\laragon\www\rnd\vendor\crocodicstudio\crudbooster\src\views\default\type_components\select2\component.blade.php) (View: C:\laragon\www\rnd\vendor\crocodicstudio\crudbooster\src\views\default\type_components\select2\component.blade.php)
I think **id_cms_privileges** were renamed in code to **cms_privileges_id** and **id_cms_menus** **to cms_menus_id** by error.

**Comment by kezern** [comment]
How can I update?
I have 5.4.0.6 version also and after php artisan crudbooster:update error is still showing.

**Comment by fherryfherry** [comment]
Update your CB version

Pada tanggal 23 Agt 2017 17.17, "kezern" <notifications@github.com> menulis:

**Comment by kezern** [comment]
I have read this doc
http://crudbooster.com/doc/5.3/upgrade-guide
It says about 5.3 version. How I upgrade in 5.4?
I suppse I have to change 
`composer require crocodicstudio/crudbooster=5.3.2.*`
to something like 5.4.* but I'm not sure

**Comment by fherryfherry** [comment]
Please use github doc only

Pada tanggal 23 Agt 2017 17.33, "kezern" <notifications@github.com> menulis:

**Comment by kezern** [comment]
Thanks I got it!

**Comment by harimayco** [comment]
Hi @fherryfherry master branch still facing this issue.
how to fix it? 
![screenshot_20170923_004606](https://user-images.githubusercontent.com/5478980/30757683-c276883e-9ff9-11e7-8788-19194c128664.jpg)

**Comment by fherryfherry** [comment]
Try to run crudbooster:update

Pada tanggal 23 Sep 2017 00.56, "Rendy Harimayco" <notifications@github.com>
menulis:

**Comment by harimayco** [comment]
@fherryfherry I already tried update command, but still facing error.
this is the 'master' branch. the error showing immidiately after login. so I cannot access anything after login.
I've checked with phpmyadmin there is no id_cms_privileges column inside cms_menus table.
![screenshot_20170925_110620](https://user-images.githubusercontent.com/5478980/30792086-cf6feb68-a1e1-11e7-8e0b-69596009e587.png)
![screenshot_20170925_110738](https://user-images.githubusercontent.com/5478980/30792088-cf74c656-a1e1-11e7-8531-60ce901cd305.png)
I want to make PR on 'master' branch but I cannot install it.
